src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs

```diff
@@ -46,7 +46,7 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                      "Source - Source location for install. Can use special 'webpi' or 'windowsfeatures' sources. Defaults to sources.",
                      option => configuration.Sources = option.remove_surrounding_quotes())
                 .Add("l|lo|local|localonly|local-only",
-                     "LocalOnly - Only search against local machine items.",
+                     "LocalOnly - Only search against local machine items. Ignores --source if provided, as local packages do not store installation source.",

```

Comment:
```suggestion
                     "LocalOnly - Only search against local machine items. Ignores --source if provided.",

```

Reply:
Given what you said above let's change this and we're done I think?

Reply:
@gep13 Any comments on this?

Reply:
Yes, I think the shorter version is better here.  There is additional work that @AdmiringWorm is doing in this area to mark certain things as deprecated in this area, as they will be removed in future versions, where `choco list` only works for locally installed packages.